src/main/java/net/dv8tion/jda/api/entities/MessageActivity.java

```diff
@@ -73,7 +73,7 @@ public String getPartyId()
      * Represents the {@link net.dv8tion.jda.api.entities.MessageActivity.Application Application} of a
      * {@link net.dv8tion.jda.api.entities.MessageActivity MessageActivity} if it's set.
      *
-     * @see <a href="https://discordapp.com/api/v7/games" target="_blank">https://discordapp.com/api/v7/games</a>
+     * @see <a href="https://discord.com/api/v7/games" target="_blank">https://discord.com/api/v7/games</a>
```

Comment:
This endpoint is gone, is there a documented replacement?

Reply:
Appears like the replacement is `https://discordapp.com/api/v6/applications/detectable`, but not documented

Reply:
That would be kinda counter-productive considering that A) The domain will change to discord.com and B) the API will update to v7.

So it would actually be https://discord.com/api/v7/applications/detectable

Reply:
Accidentally put it as discordapp.com & v6, but the endpoint is still the correct one.

Reply:
Where is this endpoint documented? If it's not documented I'd rather just remove it entirely.

Reply:
> Where is this endpoint documented? If it's not documented I'd rather just remove it entirely.

> Appears like the replacement is `https://discordapp.com/api/v6/applications/detectable`, **but not documented**

Reply:
If this isn't documented, I'll just remove it.